server/config.yaml.dist

```
@@ -3,6 +3,8 @@
 
 # shown in the website title and on the front page
 name: szurubooru
+# full url to the homepage of this szurubooru site, with no trailing slash
+domain: http://example.com
```

```suggestion
domain: # example: http://example.com
```

Probably better to leave it blank by default